biocrnpyler/chemical_reaction_network.py

```diff
@@ -29,11 +30,12 @@ class ChemicalReactionNetwork(object):
     k \Prod_{inputs i} (S_i)!/(S_i - a_i)!
     where a_i is the spectrometric coefficient of species i
     """
-    def __init__(self, species: List[Species], reactions: List[Reaction], show_warnings=False):
+    def __init__(self, species: List[Species], reactions: List[Reaction], initial_condition_dict = None, show_warnings=False):
```

Comment:
if you add a type hint here, that would help us in the long run:
```python
initial_condition_dict: Dict[Species,numbers.Real]
```
you might need to import numbers package the top of the file.

Reply:
added Dict[Species, Union[numbers.Real, Parameter]].

Reply:
Thanks!